calendar_event_type_color/views/calendar_event_type_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="view_calendar_event_type_tree" model="ir.ui.view">
+        <field name="model">calendar.event.type</field>
+        <field name="inherit_id" ref="calendar.view_calendar_event_type_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="color" widget="color_picker" optional="show" />
+            </field>
+        </field>
+    </record>
+    <record id="calendar_event_type_action" model="ir.actions.act_window">
+        <field name="name">Calendar Event Type</field>
+        <field name="res_model">calendar.event.type</field>
+        <field name="view_mode">tree</field>
+    </record>
+    <menuitem
+        id="calendar_management_menu"
+        name="Calendar Event Type"
+        parent="calendar.mail_menu_calendar"
+        action="calendar_event_type_action"
```

Comment:
```suggestion
        action="calendar_event_type_action"
        groups="base.group_system"
```

Could you add this group to the menu, to only show it to [users that have edit rights](https://github.com/odoo/odoo/blob/ba2b0020dfc2a6484fd4a9f2ed72eece414a823a/addons/calendar/security/ir.model.access.csv#L9) on the event types? 🙏🏻